bftengine/src/bcstatetransfer/SourceSelector.hpp

```diff
@@ -54,6 +57,8 @@ class SourceSelector {
   // Return true if the source should be replaced, false otherwise.
   bool shouldReplaceSource(uint64_t currTimeMilli, bool badDataFromCurrentSource) const;
 
+  bool shouldReplaceSource() const { return (currentReplica_ == currentPrimary_); }
```

Comment:
Add a TODO - should not merge to prevent mistakes. This is temporary for UT I assume?